vmngclient/session.py

```diff
@@ -246,7 +246,7 @@ def __create_base_url(self) -> str:
         scheme: str = url.scheme or "https"
         base_url = urlunparse((scheme, netloc, "", None, None, None))
         if self.port:
-            return f"{base_url}:{self.port}"
+            return f"{base_url}: {self.port}"
```

Comment:
Wait...
Isn't that breaking anything?
url will now contain space before port: `127.0.0.1 :433`

Reply:
You are right. Fix already committed